test/integration/targets/ufw/tasks/tests/basic.yml

```
@@ -20,7 +23,7 @@
   register: enable_idem_check
 - assert:
     that:
-    # FIXME - enable_check is changed
+    - enable_check is changed
     - enable is changed
     - enable_idem is not changed
     - enable_idem_check is not changed
```

After this line, could you add some random IPv4 and IPv6 rules? That will ensure that we don't have such problems again where it fails on shippable, but succeeds on other machines.

---

I'd like to do such stuff but I think I spent enough time on this PR and there is already lot of tests added.

---

No problem, I fully understand that :) This should do the trick:
```suggestion
    - enable_idem_check is not changed

- ufw:
    rule: allow
    port: 24
- ufw:
    rule: deny
    port: 25
```
At least, on my machine, the test suite still works afterwards :-) And you can get this into the PR with one click ;-)